pkg/rancher/migrations.go

```diff
@@ -0,0 +1,88 @@
+package rancher
+
+import (
+	v32 "github.com/rancher/rancher/pkg/apis/management.cattle.io/v3"
+	v3 "github.com/rancher/rancher/pkg/generated/controllers/management.cattle.io/v3"
+	rancherversion "github.com/rancher/rancher/pkg/version"
+	controllerv1 "github.com/rancher/wrangler/pkg/generated/controllers/core/v1"
+	v1 "k8s.io/api/core/v1"
+	k8serror "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	cattleNamespace                           = "cattle-system"
+	forceUpgradeLogoutConfig                  = "forceupgradelogout"
```

Comment:
`forceUpgradeLogoutConfig` and `rancherVersionKey` are not needed here.

Reply:
Done.